apps/storybook/src/VisCanvasXZoom.stories.tsx

```diff
@@ -0,0 +1,59 @@
+import { PanMesh, VisCanvas, XAxisZoomMesh } from '@h5web/lib';
+import type { ModifierKey as ModifierKeyType } from '@h5web/lib/src/vis/models';
+import type { Meta, Story } from '@storybook/react';
+
+import VisCanvasStoriesConfig from './VisCanvas.stories';
+
+interface TemplateProps {
+  disabled?: boolean;
+  modifierKey?: ModifierKeyType;
+}
+
+const Template: Story<TemplateProps> = (args) => {
+  const { disabled, modifierKey } = args;
+
+  return (
+    <VisCanvas
+      abscissaConfig={{ visDomain: [-10, 0], showGrid: true }}
+      ordinateConfig={{ visDomain: [50, 100], showGrid: true }}
+    >
+      <PanMesh />
+      <XAxisZoomMesh disabled={disabled} modifierKey={modifierKey} />
+    </VisCanvas>
+  );
+};
+
+export const Default = Template.bind({});
+Default.args = {
+  disabled: false,
+  modifierKey: 'Alt', // Initialize to default value
+};
+
+export const Disabled = Template.bind({});
+Disabled.args = {
+  disabled: true,
+  modifierKey: 'Alt', // Initialize to default value
+};
+
+export const ModifierKey = Template.bind({});
+ModifierKey.args = {
+  disabled: false,
+  modifierKey: 'Shift',
+};
+
+export default {
+  ...VisCanvasStoriesConfig,
+  title: 'Building Blocks/VisCanvas/XZoom',
```

Comment:
I'd be tempted to remove the nesting under `VisCanvas`, like for `HeatmapMesh`.

Reply:
Or a new section `PanZoom` ?

Reply:
I meant to replace the nesting `VisCanvas` section by a new section `PanZoom`

Reply:
Yep, sounds good!